debian/cephadm.postinst

```diff
@@ -28,6 +28,7 @@ case "$1" in
          adduser --quiet \
                  --system \
                  --disabled-password \
+                 --home /home/cephadm \
```

Comment:
would be ideal if we can reference the related commits in the commit message:

- 90bc0369243077c2aaf67f0de2bab5810b217f4e: pass `--home` to `adduser`
-  c5a316cfd5e993ad5accd12ed101dc33ff671163: reverts 90bc0369243077c2aaf67f0de2bab5810b217f4e, and user `~cephadm` in the place of `/home/cephadm`

so that it's more clear that to which branches we should backport this fix.

also, now that we have `#!/bin/sh` in this file's shebang line, typically, we are using dash on debian derivatives. so we can still use `~cephadm` here. so that we don't need to do the revert-pass-home dance next time as we did in the above two commits.

Reply:
> > so we can still use `~cephadm` instead of `/home/cephadm` here
> 
> will `~cephadm` work if the cephadm user doesn't exist yet?

ahh, no, it won't work. i missed where this line is used..